zsh/tests/test.bats

```diff
@@ -0,0 +1,10 @@
+expected_version="$(echo $TEST_PKG_IDENT | cut -d/ -f 3)"
+@test "zsh runs" {
+  run hab pkg exec $TEST_PKG_IDENT zsh -c 'true'
+  [ $status -eq 0 ]
+}
+
+@test "zsh --version output mentions expected version $expected_version" {
+  run hab pkg exec $TEST_PKG_IDENT zsh --version
+  [[ "$output" =~ $expected_version ]]
```

Comment:
We should, wherever possible, use single brackets, as funky behaviour can ensue. Also the regex comparision isn't necessary, we're just comparing strings.

See here for some brief information: https://github.com/sstephenson/bats/issues/49

Suggest change to:

```
[ "$output" = "$expected_version" ]
```